styles/themes/_light.scss

```diff
@@ -56,6 +56,15 @@ html.light-mode {
     color: $text-light-theme;
   }
 
+
+  .custom-navbar-item {
+    .navbar-item {
+      &:hover {
+        color: $k-hovergrey!important;
+      }
+    }
+  }
```

Comment:
IMO, we could remove these lines because they have been written on `styles/layouts/_main-navigation.scss` file.

Reply:
moved this code, thanks